tools/buildPackages.sh

```diff
@@ -0,0 +1,36 @@
+source $stdenv/setup
+
+buildPhase() {
+  echo "building archive node tools..."
+  buildArchiveDatabase
+  buildArchiveDumpDownloader
+  buildDatabaseRestoration
+  cleanup
```

Comment:
Is this invoked via Nix? If so, there's no need for 'cleanup' as the build directory is temporary.